retrieverdash/dashboard_script/dashboard_script.py

```diff
@@ -2,26 +2,45 @@
 import os
 from datetime import datetime, timezone
 from json.decoder import JSONDecodeError
-from shutil import rmtree
+from os import path, remove
+from shutil import rmtree, move
 from tempfile import mkdtemp
 
 from retriever import reload_scripts, dataset_names
 from retriever import download
 from retriever.lib.engine_tools import getmd5
 
+import sys
+import csv
 from .status_dashboard_tools import get_dataset_md5
-from .status_dashboard_tools import diff_generator
+from .status_dashboard_tools import diff_generator, diff_generator_spatial, data_shift
 from .status_dashboard_tools import create_dirs
 from .status_dashboard_tools import dataset_type
+from .status_dashboard_tools import install_postgres
 
+# To set location of the path
 file_location = os.path.normpath(os.path.dirname(os.path.realpath(__file__)))
 
-IGNORE_LIST = ['prism-climate', 'mammal-super-tree', 'forest-inventory-analysis',
+# To prevent field size errors when converting to csv
+maxInt = sys.maxsize
+decrement = True
+while decrement:
+    try:
+        csv.field_size_limit(maxInt)
+        decrement = False
+    except OverflowError:
+        maxInt = int(maxInt / 10)
+
+
+IGNORE_LIST = ['activity-timberharvest', 'prism-climate', 'mammal-super-tree', 'forest-inventory-analysis',
```

Comment:
I do not think we need this ignore list

Reply:
Let me know if we really need it, since we are running on the server.

Reply:
Ok will do that